File: cpp/23a_155A_I_Love_username.cpp
#include<bits/stdc++.h>

int main(){
    int n, small, big, temp, cnt = 0;
    std::vector<int> scores;
    std::cin>>n;
    for(int  i = 0; i < n; i++){
        std::cin>>temp;
        if (i == 0){
            scores.push_back(temp);
            small = temp;
            big = temp;
        }
        else{
            if (temp > big){
                cnt++;
                big = temp;
            }
            if ( temp < small){
                cnt++;
                small = temp;
            }
        }
    }
    std::cout<<cnt;
    return 0;
}

File: python/23a_155A_I_Love_username.py
def main():
    n = int(input())
    scores = input().split()
    for i in range(n):
        scores[i] = int(scores[i])
    small, big, cnt = scores[0], scores[0], 0
    for i in range(1, n):
        if scores[i] < small:
            cnt += 1
            small = scores[i]
        if big < scores[i]:
            cnt += 1
            big = scores[i]
    print(cnt)

if __name__ == "__main__":
    main()